### PR TITLE
[exercise-7] class photos

### PR DESCRIPTION
**Pull request by Daguis**

### CODE REVIEW

**Comment by Daguis** [comment]
la viste asi hago el merge?

File: exercises/7-class-photos/main.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#include "solutions/A-class-photos.h"
+
+using namespace std;
+
+int main()
+{
+    vector<int> redShirtHeights = { 5, 8, 1, 3, 4 };
+    vector<int> blueShirtHeights = { 6, 9, 2, 4, 5 };
+    bool expected = true;
+    auto actual = SolutionA::classPhotos(redShirtHeights, blueShirtHeights);
```
Comment (testagrossa) [inline exercises/7-class-photos/main.cpp]: replace this variable with outputA

**Review by testagrossa — CHANGES_REQUESTED** [review]

**Review by testagrossa — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -42,11 +42,10 @@ this will format all ur *.cpp and *.h files
     bazel run //exercises/4-tournament-winner:main
     bazel run //exercises/5-non-constructible-change:main
     bazel run //exercises/6-minimum-waiting-time:main
-
+    bazel run //exercises/7-class-photos:main
 
     bazel test --test_output=errors //tests:tests
 
-```
```
Comment (testagrossa) [inline README.md:49]: why removed this ?

**Review by testagrossa — CHANGES_REQUESTED** [review]

**Review by testagrossa — APPROVED** [review]

**Review by testagrossa — CHANGES_REQUESTED** [review]
CI is broken

**Pull request merged**